public/js/i18n.js

```diff
@@ -58,89 +58,91 @@ function switchLanguage(lang) {
     fileName = url_list[url_list.length - 1];
     //default file is index
     if (fileName == "") {
-	fileName = "index";
-	cur_url = cur_url + "index";
+        fileName = "index";
+        cur_url = cur_url + "index";
     }
     //check if the current page is of Chinese version
     var flag = fileName.lastIndexOf("_zh");
     if (lang === "English") {
-	createCookie(LOCAL_LANG_COOKIE, "English");
-	if ( flag > 0 ){
+        $('title').text('Crosswalk - build world class hybrid apps');
```

Comment:
This is the only line that really changed in this file, right?  What behaviour does it change?  I tried switching back and forth on the published site and the title looks fine.